tests/python/unittest/test_random.py

```diff
@@ -717,12 +717,24 @@ def test_random_seed():
     shape = (5, 5)
     seed = rnd.randint(-(1 << 31), (1 << 31))
     mx.random.seed(seed)
-    v1 = mx.nd.random_uniform(shape=shape)
+
+    def _assert_same_mx_arrays(a, b):
+        assert len(a) == len(b)
+        for a_i, b_i in zip(a, b):
+            assert (a_i.asnumpy() == b_i.asnumpy()).all()
+
+    N = 100
+    v1 = [mx.nd.random_uniform(shape=shape) for _ in range(N)]
```

Comment:
nit: `mx.random.uniform` is the preferred way now.

Reply:
Updated. Thank you!